src/components/Layouts/ReportLayout.less

```diff
@@ -63,7 +63,7 @@ h1 {
       overflow: visible;
       transition: none;
       z-index: 1;
-      height: auto !important;
+      height: auto;
```

Comment:
i think this is used for the layout fixes. have you checked out all test cases?

Reply:
tested several cases, can you point me to specific test cases?

Reply:
lets revert this change for now